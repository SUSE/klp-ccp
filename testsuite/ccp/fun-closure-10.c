int pu_f(void)
{
	g();
}
/* cp-out: warning: [^:]*: line 3, column 1: identifier "g" not declared */
