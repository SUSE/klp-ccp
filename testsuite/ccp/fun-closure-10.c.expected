int klpp_pu_f(void)
{
	g();
}
